jest.config.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I moved the `jest.config.js` to `jest.config.json` just in order to be consistent with prettier and eslint configs as JSON. The prettier config in particular doesn't seem to work properly if you change to the JS module style. (There is some issue with it not being able to detect the TS `parserOptions` correctly and it just gives an error.)